modin/experimental/core/execution/native/implementations/hdk_on_native/test/test_dataframe.py

```diff
@@ -939,7 +939,20 @@ def groupby(df, **kwargs):
         run_and_compare(groupby, data=self.data)
 
     @pytest.mark.parametrize("by", [["a"], ["a", "b", "c"]])
-    @pytest.mark.parametrize("agg", ["sum", "size", "mean"])
+    @pytest.mark.parametrize(
+        "agg",
+        [
+            "sum",
+            "size",
+            "mean",
+            pytest.param(
+                "median",
+                marks=pytest.mark.xfail(
+                    reason="https://github.com/intel-ai/hdk/pull/506"
```

Comment:
This PR has been merged.

Reply:
Yeah waiting when it’s released :)

Reply:
The test is xfailed until the fix is released, but the PR could already be merged. It would allow using this functionality with HDK builds from the main branch.

Reply:
@AndreyPavlenko If we merge it, will there be an error instead of slow execution?

Reply:
Yes, there will be an error when running on the latest HDK release. Currently, it defaults to pandas.